ksqldb-engine/src/main/java/io/confluent/ksql/planner/plan/FinalProjectNode.java

```diff
@@ -126,21 +125,6 @@ public void validateKeyPresent(final SourceName sinkName) {
         }
         return false;
       });
-
-      for (final Entry<ColumnName, Set<ColumnName>> seenKey : seenKeyColumns.entrySet()) {
```

Comment:
Intellij marked this code as "dead" because `seenKeyColumns.entrySet()` is always empty.